### DISCUSSION TITLE
TAILWIND CSS DONT WORK WITH VERCEL DEPLOY HELP PLS

### DISCUSSION THREAD

**Discussion by LisandroGG** [tailwindlabs/tailwindcss#11835] [Help] [answered]
A few days ago I am having the problem that my deploy does not show the tailwind styles, I leave here my repository and my deploy

thank you

https://portfolio-lake-alpha-89.vercel.app/
https://github.com/LisandroGG/Portfolio

**Comment by wongjn** [comment]
You seem to have windicss installed, do you mean to be using that or are you intending to use Tailwind?

You seem to be missing a `postcss.config.js` file in the root of the project to run Tailwind. It should have a minimum of

```js
module.exports = {
  plugins: {
    tailwindcss: {},
  },
}
```

**Reply by LisandroGG** [reply]
I'm using tailwind, and to group styles on hover, active and others, I use windicss

**Reply by wongjn** [reply]
Thank you for the clarification. Consider adding the `postcss.config.js` as mentioned previously.

**Reply by LisandroGG** [reply]
would the postcss.config.js file look like this?

default export {
    accessories: {
      tailwind: {},
      auto prefix: {},
    }
  }

while the tailwind.config.js like this?

/** @type {import('tailwindcss').Config} */
default export {
  mode: 'jit',
  content: ["./index.html", "./src/**/*.{html,js,jsx,ts,tsx}", "./components/**/*.{js,ts,jsx ,tsx}"],
  issue: {
    colors: {
      
    },
    extend: {
      
    },
  },
  accessories: [],
}

**Reply by wongjn** [reply]
As expressed previously, the `postcss.config.js` file should have a minimum of

```js
module.exports = {
  plugins: {
    tailwindcss: {},
  },
}
```

**Reply by LisandroGG** [reply]
That solution would not be working for me, what could I do?

Could you pass me the configuration of the files tailwind.config.js, postcss.config.js and vite.config.js?

sorry for the inconvenience

**Reply by wongjn** [reply, marked as answer]
You'd probably want to remove the empty `colors` object, and the obsolete `mode` key in `tailwind.config.js`:

```diff
-mode: 'jit',
 content: ["./index.html", "./src/**/*.{html,js,jsx,ts,tsx}", "./components/**/*.{js,ts,jsx,tsx}"],
 theme: {
-  colors: {
-    
-  },
   extend: {
```

Otherwise, `postcss.config.js` and `vite.config.js` look adequate enough.

**Reply by LisandroGG** [reply]
Thank you man, you saved me :D

**Reply by hasnahadd** [reply]
hey i have the same issue most of the colors position of element not working in deployment  this is the postcss,mjs  config  /** @type {import('postcss-load-config').Config} */
const config = {
  plugins: {
    tailwindcss: {},
  },
};

export default config; and the tailwind config file  /** @type {import('tailwindcss').Config} */
module.exports = {
  content: [
    "./pages/**/*.{js,ts,jsx,tsx,mdx}",
    "./components/**/*.{js,ts,jsx,tsx,mdx}",
    "./app/**/*.{js,ts,jsx,tsx,mdx}",
  ],
  theme: {
    extend: {
      backgroundImage: {
        "gradient-radial": "radial-gradient(var(--tw-gradient-stops))",
        "gradient-conic":
          "conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))",
      },
      colors: {
        magenta: '#c922a8',
        turquoise: '#72e8e9',
      },
    },
  },
  plugins: [],
};

this is the deployment  website    [deployment ](https://q-rgenerator-two.vercel.app/)
and this is my repo   [repo](https://github.com/hasnahadd/QRgenerator)
